modules/installation-azure-user-defined-routing.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-private.adoc
+
+[id="installation-azure-user-defined-routing_{context}"]
+= User-defined outbound routing
+
+In {product-title}, you can choose your own outbound routing for a cluster to
+connect to the internet. This allows you to skip the creation of public IP
+addresses and the public load balancer.
+
+You can configure user-defined routing by modifying parameters in the
+`install-config.yaml` file before installing your cluster. A pre-existing VNet
+is required to use outbound routing when installing a cluster; the installation
+program is not responsible for configuring this.
+
+When configuring a cluster to use user-defined routing, the installation program
+does not create the following resources:
+
+* Outbound rules for access to Internet.
+* Public IPs for the public load balancer.
+* Kubernetes Service object to add the cluster machines to the public load
+balancer for outbound requests.
+
+You must ensure the following items are available before setting user-defined
+routing:
+
+* Egress to the internet is possible to pull container images, unless using an
+internal registry mirror.
+* The cluster can access Azure APIs.
+* Various allowlist endpoints are configured. You can reference these endpoints
+in the *Configuring your firewall* section.
```

Comment:
I know we have to work around stuff like this because we can't link. I'm not a fan of doing this implicit dependency, but if you do want to keep it, I'd make the section italic, not bold. Bold is usually reserved for GUI items.